lib/internal/Magento/Framework/GraphQl/Query/ResolverInterface.php

```diff
@@ -20,7 +20,7 @@
      * Fetches the data from persistence models and format it according to the GraphQL schema.
      *
      * @param \Magento\Framework\GraphQl\Config\Element\Field $field
-     * @param $context
+     * @param \Magento\Framework\GraphQl\Query\Resolver\ContextInterface $context
```

Comment:
Is fixed in 2.3.0 performance improvements and will result in a conflict. Better revert it here.